code/game/objects/items/devices/telecrystal.dm

```diff
@@ -0,0 +1,19 @@
+/* 
+Telecrystal item
+Does nothing if not suitable antag type, checks for accept_tcrystals = 1 in a mob's mind.
+For new antags, make sure to add "player.mind.accept_tcrystals = 1" if you want tradable tcrystals.
+*/
+/obj/item/device/telecrystal
+	name = "Red crystal"
```

Comment:
Will be swapped to lowercase. Didn't run into any problems operating the item, but makes sense.